spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -110,10 +116,17 @@ You can use the command `kubectl get all -l app=metrics` to verify that the depl
 +
 . Deploy Skipper
 +
-This is an optional step.  Deploy link:http://cloud.spring.io/spring-cloud-skipper/[Skipper] if you want the added features of upgrading and rolling back Streams since Data Flow delegates to Skipper for those features.
+Deploy link:http://cloud.spring.io/spring-cloud-skipper/[Skipper] to leverage the features of upgrading and rolling back Streams since Data Flow delegates to Skipper for those features.
```

Comment:
I don't believe we should request users to deploy skipper as a mandatory step for 1.3. I'm aware it's currently failing on k8s at least, but I thought it was a bug on our end. For 1.3 at least it should still be optional.

Reply:
As discussed in slack, it follows the same design as Metrics server. Today. the [server-deployment.yaml](https://github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes/blob/master/src/kubernetes/server/server-deployment.yaml#L52-L56) expects to resolve the metrics and skipper-server endpoints at the deployment time. 

If we want them to be truly optional, we need to come up with a different approach to resolve the endpoints.

As an aside, Skipper will be the preferred deployment model for streaming use-cases in SCDF going forward, so I think it can be included in the getting-started guide (starting from 1.3 at least). We want the users to start getting used to it and leverage the benefits it brings.

Let's also hear from @markpollack / @trisberg on their perspectives.

Reply:
Whatever we decide on this front, we need to also make sure the CF-docs are updated for consistency. Currently, the [getting-started](https://docs.spring.io/spring-cloud-dataflow-server-cloudfoundry/docs/1.3.0.M3/reference/htmlsingle/#getting-started-cloudfoundry) guide includes Skipper provisioning step as well.

Reply:
Based on the discussion with @markpollack, we have decided to comment Skipper's URI k/v pair from the YAML. If someone wants it, they can uncomment before running the `kubectl create -f src/kubernetes/server/server-deployment.yaml` command. 

SCDF's helm-chart experience, however, by default would include Skipper component in it. The chart will be updated once the 1.3 GA release is available.

I will submit another commit clarifying this position.